scss/_variables.scss

```diff
@@ -1,32 +1,8 @@
-// Blues
-$blue: #4895d9; // old blue: #6da3d3
-$blue-lightest: lighten($blue, 33%);
-$blue-light: lighten($blue, 20%);
-
-$blue-active-highlight: lighten($blue, 10%);
-
-$blue-darkest: darken($blue, 20%);
-
-// Grays
-$gray: #808080;
-
-$gray-lightest: lighten($gray, 38%);
-$gray-darkest: darken($gray, 38.5%);
-
-// $gray color functions from light to dark:
-//
-// lighten($gray, 30%)
-// lighten($gray, 20%)
-// lighten($gray, 10%)
-// darken($gray, 10%)
-// darken($gray, 20%)
-// darken($gray, 30%)
-
-// Secondary
+// Secondary colors
 $yellow: #f0b849;
```

Comment:
Can we transition the rest of these colors to color studio? A separate PR may be best given the size of this PR as it stands.

Reply:
Yeah, these aren't used in many places so I can update them. I wasn't sure if we wanted to use color studio for the secondary stuff like greens.